tests/cpp/operator/mkldnn_test.cc

```diff
@@ -100,9 +100,9 @@ static void VerifyDefMem(const mkldnn::memory &mem) {
 
 TEST(MKLDNN_UTIL_FUNC, MemFormat) {
   // Check whether the number of format is correct.
-  CHECK_EQ(mkldnn_format_last, 67);
-  CHECK_EQ(mkldnn_nchw, 5);
-  CHECK_EQ(mkldnn_oihw, 15);
+  CHECK_EQ(mkldnn_format_last, 110);
+  CHECK_EQ(mkldnn_nchw, 7);
```

Comment:
Is there any way we could use enums or consts vars somewhere that would be able to describe what the integers represent here?  It would be interesting for me to see what memory format 67 used to represent, and the new format 110 means.

Reply:
Hi @KellenSunderland , data formats are in an enumerate type of MKL-DNN and may be changed in the evolution of MKL-DNN. This test cast is trying to remind the developer who submit PR to update MKL-DNN version in MXNet to take care of the format change of MKL-DNN.